downloads/pt/github-git-cheat-sheet.md

```diff
@@ -0,0 +1,247 @@
+---
+layout: cheat-sheet
+title: Folheto de ajuda para Git do GitHub (pt-BR)
+byline: Git é um sistema de controlo de versão distribuído open source que facilita ações com o GitHub no seu notebook ou desktop. Este folheto de ajuda resume instruções frequentemente usadas na linha de comando do Git para referência rápida.
+leadingpath: ../../
+---
+
+{% capture colOne %}
+## Instalar o Git
+
+O GitHub fornece clientes desktop que incluem um interface gráfico para as ações mais comuns em repositórios. Estes clientes também incluem ferramentas da linha de comandos para cenários mais avançados. Em ambos os casos, os clientes são actualizados automáticamente.
+
+### GitHub para Windows
+[http://windows.github.com](http://windows.github.com)
+
+### GitHub para Mac
+[http://mac.github.com](http://mac.github.com)
+
+Distribuições de Git para Linux e sistemas POSIX estão disponíveis no site oficial do Git SCM.
+
+### Git para todas as plataformas
+[http://git-scm.com](http://git-scm.com)
+
+## Configurar o Git
+
+Configurar o nome que ficará associado aos Git commits
+
+```$ git config --global user.name "[nome]"```
+
+Configurar o email que ficará associado aos Git commits
+
+
+```$ git config --global user.email "[endereco-de-email]"```
+
+
+## Criar repositórios
+Inicie um novo repositório ou obtenha de uma URL existente
+
+
+```$ git init [nome-do-projeto]```
+
+Cria um novo repositório local com um nome especificado
+
+
+```$ git clone [url]```
+
+Baixa um projeto e seu histórico de versão inteiro
```

Comment:
Ok, this was lost in the translatation, "baixar" is totally a nope:

![baixar](http://i.giphy.com/3osxYoiN5ElORrgzN6.gif)